docs/src/modules/concepts/pages/multi-region.adoc

```diff
@@ -48,16 +70,16 @@ image:geo-e.svg[Geo data replication, width=600]
 
 Meanwhile, all requests made by Alice on her data are handled locally image:steps-1.svg[width=20] and get an immediate reply image:steps-2.svg[width=20]. The write operations are being replicated to the UK image:steps-3.svg[width=20].
 
-=== Primary selection
+== Replicated writes
 
-How Akka assigns the primary region to an entity is configurable. The two main modes are **static**, and **dynamic**.
+The replicated write replication strategy allows every region to be capable of handling writes for all entities. This is done through the use of CRDTs, which can be modified concurrently in different regions, and their changes safely merged without conflict.
 
-In the **static primary selection** mode (which is the default), the primary region for an entity is selected statically as part of the deployment, so all entities have the same primary region. This is useful for scenarios where you want one primary region, with the ability to fail over to another region in the case of a regional outage.
+== Replication Guarantees
 
-In the **dynamic primary selection** mode, each entity can have a different region that is considered its primary region. This is selected based on whichever region the entity was first written in. This is useful for scenarios where you want to have the primary region for you data close to the users who use the data. A user, Alice, in the USA, will have her data in the USA, while a user Bob, in the UK, will have his data, in the UK.
+Akka guarantees that all events created within one region are eventually replicated to all other regions in the project.
 
-The Operating section explains more details about xref:operations:regions/index.adoc#selecting-primary[configuring the primary selection mode].
+Each entity’s state is a series of events that are persisted in a local event journal, which acts as the source of events that must be replicated from one region to another. Having a durable, local event journal is the foundation for how Akka can recover an entity’s state in the event of failure.
 
-== Replicated writes
+Each event has a sequence number that is validated on the receiving side to guarantee correct ordering and exactly-once processing of the events. A replicated event is processed by entities in other regions by having it added to its local event journal. Once added to the local event journal, it can be used to handle read requests in those regions.
```

Comment:
The ambiguous `it`s make this paragraph a little more difficult to read. And maybe clarify the last `it` to being not just the event, but the derived state. Suggestion:

```suggestion
Each event has a sequence number that is validated on the receiving side to guarantee correct ordering and exactly-once processing of the events. A replicated event is processed by entities in other regions by having the event added to the local event journal of each entity. Once added to the local event journal, the replicated event can be used to update the entity's state and handle read requests in those regions.
```